src/views/Timesheets/index.jsx

```diff
@@ -39,7 +39,7 @@ const withNoSeconds = (date) => set(date, { seconds: 0, milliseconds: 0 });
 
 const CustomTooltip = withStyles((theme) => ({
   tooltip: {
-    backgroundColor: theme.palette.common.black,
+    backgroundColor: theme.colorSchemes.light.palette.common.black,
```

Comment:
As above, we shouldn't merge anything that hard-codes the light theme.

Reply:
Reverting for now :+1: